src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to the SCSS file is incorrect. In an HTML document, you should link to a CSS file. Make sure to compile your SCSS to CSS and link to the resulting CSS file instead.